### ISSUE TITLE
Implementing ChemProp Model architecture in DGL-LifeSci

### ISSUE THREAD

**Issue by leelasd** [awslabs/dgl-lifesci#195]
Hi @mufeili, 

I was wondering if there are any plans to implement ChemProp model in DGL-LifeSci. It is a Directional-MPNN, and since DGL-LifeSci already has MPNN networks how complicated do you think it would be for a novice like me to implement this in DGL? 

Leela

**Comment by mufeili** [comment]
Hi @leelasd ,

I checked D-MPNN long before and no longer remember it clearly. I can check that when I have time. Alternatively, could you summarize the differences between MPNN and D-MPNN?
